Userland/Libraries/LibJS/Runtime/Temporal/CalendarPrototype.cpp

```diff
@@ -54,6 +57,39 @@ JS_DEFINE_NATIVE_FUNCTION(CalendarPrototype::id_getter)
     return js_string(vm, calendar.to_string(global_object));
 }
 
+// 12.4.4 Temporal.Calendar.prototype.dateFromFields ( fields, options )
```

Comment:
```suggestion
// 12.4.4 Temporal.Calendar.prototype.dateFromFields ( fields, options ), https://tc39.es/proposal-temporal/#sec-temporal.calendar.prototype.datefromfields
// NOTE: This is the minimum dateFromFields implementation for engines without ECMA-402.
```

to make it clear that there's a second implementation that needs to be used when ECMA-402 is implemented.

Reply:
done